template/partials/head.tmpl.partial

```diff
@@ -0,0 +1,46 @@
+{{!Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license.}}
+
+<head>
```

Comment:
Just nothing that this override means we need to react if there are any upstream changes in the modern template... This is what caused things when I was upgrading things, which is why I removed it.

I'm hoping maybe there's some way to just add our favicons rather than replacing the entire `<head>`, which seems like it could cause similar trouble in the future... Is something like that possible, like some more targeted partial or extensibility hook specifically for favicon?

Reply:
> I'm hoping maybe there's some way to just add our favicons rather than replacing the entire <head>

Not out of the box. That would be an upstream PR to docfx.